pkg/plugins/discovery.go

```diff
@@ -22,41 +22,39 @@ type Registry interface {
 	Plugin(name string) (*Plugin, error)
 }
 
-type LocalRegistry struct {
-	path string
-}
-
-func newLocalRegistry(path string) *LocalRegistry {
-	if len(path) == 0 {
-		path = defaultLocalRegistry
-	}
+type LocalRegistry struct{}
 
-	return &LocalRegistry{path}
+func newLocalRegistry() LocalRegistry {
```

Comment:
yeah, I had the same feeling from the beginning, but naming is hard :smile_cat: 

Let's think about something better somewhere else.